src/symbolic.jl

```diff
@@ -0,0 +1,88 @@
+# config for build bert
+const config = Dict(
+  "attention_probs_dropout_prob" => 0.1,
+  "initializer_range"            => 0.02,
+  "hidden_act"                   => gelu,
+  "num_hidden_layers"            => 12,
+  "hidden_size"                  => 768,
+  "max_position_embeddings"      => 512,
+  "type_vocab_size"              => 2,
+  "vocab_size"                   => 30522,
+  "num_attention_heads"          => 12,
+  "hidden_dropout_prob"          => 0.1,
+  "intermediate_size"            => 3072,
+)
+
+# create bert model like pretrain struct
+# you can define the bert model in the way you like and wrap it with TransformerModel
+function create_bert()
+    global config
```

Comment:
I don't get this config dictionary. Why not just have these be keyword arguments to `create_bert`?

Reply:
Hi Chris, this is just a starter boilerplate from Internet. I will mark ready for review when you can feedback.

Reply:
oh okay, no worries.